### ISSUE TITLE
too many voices needed for one voice activation

### ISSUE THREAD

**Issue by AlyxZiel** [MYOHA-1/CeVIO-AI-Bypass#2]
I only need to use POPY and ROSE, do I really need to install every voice from Kafu to Minato just so I can use them both? It takes too much storage space (I've only installed Kafu and IA but they work fine) so far that's my only issue on this crack

**Comment by gandam21** [comment]
> I only need to use POPY and ROSE, do I really need to install every voice from Kafu to Minato just so I can use them both? It takes too much storage space (I've only installed Kafu and IA but they work fine) so far that's my only issue on this crack

Do I need to install the editor and voice first and turn on the crack to work?

**Comment by AlyxZiel** [comment]
> > I only need to use POPY and ROSE, do I really need to install every voice from Kafu to Minato just so I can use them both? It takes too much storage space (I've only installed Kafu and IA but they work fine) so far that's my only issue on this crack
> 
> Do I need to install the editor and voice first and turn on the crack to work? I Only need KZN and 3 Tohoku,and sasara,tsuzumi,minato.

Yes, you still need to install almost all of them so you can use the last one (in your case, the last is Minato, so from KAFU to Minato, you need to install ALL voices.) ((KAFU->IA->KZN->Sekai->ONE->Sasara->Yukari Rei->Tohoku Kiritan->Tohoku Itako->RIME->Tohoku Zunko->Tsudumi->Minato->POPY->ROSE))

I want to know if there's an actual way to install just the selected voices without activating the others, because it takes up too much space
